front/components/poke/data_sources/view.tsx

```diff
@@ -105,8 +118,8 @@ export function ViewDataSourceTable({
                 {connector && (
                   <>
                     <PokeTableRow>
-                      <PokeTableCell>Is Running?</PokeTableCell>
```

Comment:
I see this as redline meaning that I don't see iPaused anymore no?

Reply:
You are confused by the diff.

I looks like this:
Is Running: ❌|✅

And below:
Paused at: N/A|date.

I can add a more explicit Is Paused: ❌|✅ if you feel that it's necessary ? (do you want me to ?)

Reply:
OK! Missed that paused was basically repeated in two places :+1: